Apps/AdminWebClient/src/Server/Controllers/EmailAdminController.cs

```diff
@@ -76,24 +69,31 @@ public IActionResult GetEmails()
         }
 
         /// <summary>
-        /// Creates a clone of the email id and requeues it for sending.
+        /// Creates a clone of the emails and requeues it for sending.
         /// </summary>
-        /// <returns>An ok result unless the email id is not foun.</returns>
-        /// <param name="emailId">The email Id to resend.</param>
-        /// <response code="200">Returns the list of beta requests.</response>
+        /// <returns>An ok result unless the email id is not found.</returns>
+        /// <param name="emailIds">The email Ids to resend.</param>
+        /// <response code="200">Successfully queued all emails.</response>
         /// <response code="401">the client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpPost]
-        [Route("{emailId}")]
-        public IActionResult ResendEmail(Guid emailId)
+        public IActionResult ResendEmail(List<Guid> emailIds)
         {
+            if (emailIds == null)
+            {
+                throw new ArgumentNullException(nameof(emailIds));
```

Comment:
Instead of throwing an exception we should probably return a 400 or 418 HTTP Error.